### DISCUSSION TITLE
Duda sobre output parte 1

### DISCUSSION THREAD

**Discussion by lfb1206** [IIC2133-PUC/2024-1#160] [Tarea 2] [answered]
Hola!

Me entro una duda sobre el output en esta parte, esto porque en el enunciado se dice "Luego, separado en líneas
distintas, las posiciones de cada cadena en el documento" pero en ningún test se revisa que una consulta se encuentre dos veces en la cadena.

Entonces mi pregunta es, se debe encontrar una posición o todas? Porque encontrar una disminuye bastante la complejidad del código y no se está evaluando encontrar todas.

Gracias de antemano.

**Comment by Monti7475** [comment]
En el output del test easy-2 se ve que se retornan dos valores por una consulta
![image](https://github.com/IIC2133-PUC/2024-1/assets/130416325/a05afd1d-6695-4c13-9afe-70fadfbc145b)
Que consiste en buscar CC en AGCCCGC
Por lo que diría que si se buscan todas

**Reply by lfb1206** [reply]
Me ilusioné, pero parece que tienes otros tests :(, porque los actualizados no tienen ese formato en el easy-2, los acabo de descargar.

**Reply by Monti7475** [reply]
Oh tienes razón
Aunque en el enunciado aparece un ejemplo con doble output (dos match)
Asumo que si los hubiesen cambiado por eso, hubiesen cambiado el enunciado también, por lo que yo diría que debería seguir así
Igual mejor esperar a que un ayudante vea este discussion :(

**Comment by sachondo7** [comment, marked as answer]
Efectivamente, puede haber más de un valor por consulta

**Reply by lfb1206** [reply]
¿Y como se chequea si se está cumpliendo con esto? Porque, como dije previamente, no hay tests que lo chequee.

**Reply by sachondo7** [reply]
Toco la mala suerte que el generador de tests hizo casos sin múltiples resultados. Vamos a intentar regenerarlos. 
Igualmente, los tests que se suben son solo referenciales, por lo que si quieres probar que tu código funciona con múltiples resultados te recomiendo agregar manualmente casos en tus archivos de input (con su respuesta correcta en el output) para que así lo puedas probar.
